src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -126,18 +128,21 @@
             icon]])))]))
 
 (defn reactions-and-actions
-  [{:keys [message-id outgoing-status deleted? deleted-for-me?] :as message-data}
+  [{:keys [message-id deleted? deleted-for-me?] :as message-data}
    {:keys [chat-id] :as context}]
   (fn []
-    (let [actions        (get-actions message-data context)
-          main-actions   (filter #(= (:type %) :main) actions)
-          danger-actions (filter #(= (:type %) :danger) actions)
-          admin-actions  (filter #(= (:type %) :admin) actions)]
+    (let [data            (if (contains? message-data :album-id)
+                            (first (:album message-data))
+                            message-data)
```

Comment:
I think it's better to move the destructuring of `message-data` after `data`, it will draw more attention when others extract more value from `data` in the future